extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/util/MultiStageQueryContext.java

```diff
@@ -213,6 +215,11 @@ public static IndexSpec getIndexSpec(final QueryContext queryContext, final Obje
     return decodeIndexSpec(queryContext.get(CTX_INDEX_SPEC), objectMapper);
   }
 
+  public static boolean useAutoColumnSchemas(final QueryContext queryContext)
```

Comment:
Please document the use of this externally undocumented feature in the class level docs. https://github.com/apache/druid/pull/14175/files#diff-cbcc2f2c9d571876786fe63eef35c2d82d64de0700334267c22f83f31d633af0L51